chain/client/src/client.rs

```diff
@@ -707,6 +737,7 @@ impl Client {
         // TODO: replace to channels or cross beams here? we don't have multi-threading here so it's mostly to get around borrow checker.
         let accepted_blocks = Arc::new(RwLock::new(vec![]));
         let blocks_missing_chunks = Arc::new(RwLock::new(vec![]));
+        let orphans_missing_chunks = Arc::new(RwLock::new(vec![]));
```

Comment:
fyi I believe the use of `Arc<RwLock<..>>` here is completely unnecessary and can be simply replaced by `Rc<RefCell<..>>`. It was written at a time when Rust was still foreign to us...

Reply:
I'll create an issue that tracks this